internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -874,6 +878,20 @@ func (c *ClusterManager) buildObservatoriumDexSecretResource() *k8sCoreV1.Secret
 	}
 }
 
+func (c *ClusterManager) buildDataplaneRhssoCredentialsSecretResource(cluster *api.Cluster) *k8sCoreV1.Secret {
+	return &k8sCoreV1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "observability-proxy-credentials",
+			Namespace: observabilityNamespace,
+		},
+		StringData: map[string]string{
+			"client_id":             cluster.ClientID,
+			"client_secret":         cluster.ClientSecret,
+			"rhsso_auth_server_url": c.ObservabilityConfiguration.RedHatSsoAuthServerUrl,
```

Comment:
Thanks for adding this.
However the value `c.ObservabilityConfiguration.RedHatSsoAuthServerUrl` only works if the sso provider type is `redhat_sso`. We could make this a bit generic and do something like what's done here https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/36c3940df8b8d5deb312bbeadee8dd6c22ccaf5b/internal/kafka/internal/services/kas_fleetshard_operator_addon.go#L167
i.e get this value from the keycloak config.

Reply:
That's done now.